src/hooks/useDateViewType.ts

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+
+export const useDateViewType = () => {
```

Comment:
it probably can be named 'useClickSwitch' or similar because it doesn't do anything related to time here.
just return values as an array to allow custom naming